skins/laika/src/components/pages/donation_form/Address.vue

```diff
@@ -98,6 +95,12 @@ export default Vue.extend( {
 					pattern: '',
 					optionalField: false,
 				},
+				email: {
+					name: 'email',
+					value: '',
+					pattern: '^[^@]+@.+$',
+					optionalField: false,
+				},
```

Comment:
You could take the opportunity and get rid of the `setEmail` action in the store and use the `setAddressField` action instead. Besides more concise code, that has the benefit that the email will be validated with the pattern.